### PR TITLE
Update polybase-configure-teradata.md

### PR DESCRIPTION
**Pull request by pabechevb**

### CODE REVIEW

**Comment by PRMerger8** [comment]
@pabechevb : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by MikeRayMSFT** [comment]
@pabechevb - thank you for the submission. We are reviewing.

File: docs/relational-databases/polybase/polybase-configure-teradata.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ To query the data from a Teradata data source, you must create external tables t
 The following Transact-SQL commands are used in this section:
 
 - [CREATE DATABASE SCOPED CREDENTIAL (Transact-SQL)](../../t-sql/statements/create-database-scoped-credential-transact-sql.md)
-- [CREATE EXTERNAL DATA SOURCE (Transact-SQL)](../../t-sql/statements/create-external-data-source-transact-sql.md) 
+- [CREATE EXTERNAL DATA SOURCE (Transact-SQL)](../../t-sql/statements/create-external-data-source-transact-sql.md)
+- [CREATE EXTERNAL TABLE (Transact-SQL)](../../t-sql/statements/create-external-table-transact-sql.md)
 - [CREATE STATISTICS (Transact-SQL)](../../t-sql/statements/create-statistics-transact-sql.md)
 
 1. Create a database scoped credential for accessing the MongoDB source.
```
Comment (NathanMSFT) [inline docs/relational-databases/polybase/polybase-configure-teradata.md:41]: This should read "Teradata source" instead of "MongoDB source".

**Review by NathanMSFT — CHANGES_REQUESTED** [review]
These changes look good. I also suggested to make a change to the "Create a database scoped credential" sentence (#1 point)

**Comment by PRMerger20** [comment]
@pabechevb : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
Moving to private repo for staging.

**Pull request closed without merging**